doc/intl/de_GER/README.md

```diff
@@ -0,0 +1,104 @@
+
+| Funktion | mainnet | testnet | regtest |
+| :------- | ------: | ------: | ------: |
+| P2P      |   22556 |   44556 |   18444 |
+| RPC      |   22555 |   44555 |   18332 |
+
+## Laufende Entwicklungen - Fahrplan zum Mond 🌒
+
+Dogecoin Core ist eine Open-Source Software und wird von der Community entwickelt.
+Der Entwicklungsprozess ist transparent und öffentlich Einsehbar; Jeder kann ihn sehen, darüber diskutieren und daran teilhaben!
+
+Die wichtigsten Entwicklungsressourcen:
+
+* [GitHub Projekte](https://github.com/dogecoin/dogecoin/projects) 
+  Wird verwendet, um den bereits geplanten und derzeit in Entwicklung befindlichen Releases zu folgen.
+* [GitHub Diskussionen](https://github.com/dogecoin/dogecoin/discussions)
+  Wird verwendet, um geplante und ungeplante Funktionen (Features) zu diskutieren, die sich auf die Entwicklung der Dogecoin Core-Software, die zugrunde liegenden Protokolle und das DOGE-Asset beziehen.
+* [Dogecoindev subreddit](https://www.reddit.com/r/dogecoindev/)
+
+### Versionsstrategie
+
+Die Vergabe der Versionsnummern efolgt nach dem Prinzip von ```major.minor.patch```.
+
+### Branches
+Es gibt 4 Arten von "Branches" in diesem Repository:
+
+- **master:** Instabil - Enthält den neuesten Code, der sich derzeit in der Entwicklung befindet.
+- **maintenance:** Stabil, enthält die neueste Version früherer Versionen, die noch aktiv gewartet werden. Format: ```<version>-maint```
+- **development:** Intabil, enthält neuen Code für kommende Versionen. Format: ```<version>-dev```
+- **archive:** Stabile, unveränderliche Branches für alte Versionen, die sich nicht mehr ändern, weil sie nicht mehr gepflegt werden.
+
+***Reichen Sie Ihre Pull-Requests im `master`-Branch ein!***
+
+* Die Wartungs-Branches sind ausschließlich durch Freigabe veränderbar. Wenn eine Veröffentlichung geplant ist, wird ein Entwicklungs-Branch erstellt und die Beitrage ("Commits") werden von den Betreuern aus dem Master-Branch ausgewählt.
+
+## Beitragen 🤝
+ 
+Falls Sie einen Fehler oder Probleme beim Verwenden dieser Software finden, melden Sie diese über das vorhandene [Ticket System](https://github.com/dogecoin/dogecoin/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5Bbug%5D+).
+ 
+Wenn Sie an der Mithilfe oder der Entwicklung von Dogecoin Core interessiert sind sehen Sie sich Bitte den [Beitragsleitfaden](CONTRIBUTING.md) an.
+Oft gibt es Themen, die [Hilfe](https://github.com/dogecoin/dogecoin/labels/help%20wanted) benötigen. Ihr Beitrag könnte einen hohen Stellenwert haben und wird daher auch sehr geschätzt.
+ 
+## Communities 🚀🍾
+ 
+Sie können den Communities auf verschiedenen sozialen Medien beitreten.
+Herzlich sind sie Eingeladen sich umzusehen um neue Leute kennenzulernen, die neueste Memes zu teilen, um Hilfe anzubieten oder auch zu erhalten und an Diskussionen teilzuhaben.
+Vielleicht findet sich ja auch Jemand, der Ihnen bei ihren Projekten behilflich sein kann.
```

Comment:
```suggestion
Vielleicht findet sich ja auch jemand, der Ihnen bei Ihren Projekten behilflich sein kann.
```

Lowercased "Jemand" and changed "ihren" to "Ihren" to match the capitalization and case correctly.